p2p/sentry/sentry_multi_client/sentry_multi_client.go

```diff
@@ -158,7 +153,15 @@ func SentryReconnectAndPumpStreamLoop[TMessage interface{}](
 			continue
 		}
 
-		if _, err := sentry.SetStatus(ctx, statusDataFactory()); err != nil {
+		statusData, err := statusDataFactory(ctx)
+		if err != nil {
+			logger.Error("SentryReconnectAndPumpStreamLoop: statusDataFactory error", "stream", streamName, "err", err)
+			if statusData == nil {
```

Comment:
what is the value of this if statusData == nil { panic(...) }?
instead of this I think whenever err != nil we should do 
```
logger.Error(...)
time.Sleep(time.Second)
continue
```

similar to how handle SetStatus

Reply:
it is actually bad if you panic here since you will lose the goroutine and there is no `recover` here meaning the node will be just left disconnected

Reply:
note statusData may return intermittent errors since it is a DB call (in our case maybe highly unlikely due to how mdbx works but the code here should be ok with a DB transient error)